Traceback (most recent call last):
  File "/deac/csc/khuriGrp/zhaok220/clustering_1/scripts/pop_iter_scrna.py", line 4, in <module>
    from clusterer import *
  File "/deac/csc/khuriGrp/zhaok220/clustering_1/scripts/clusterer.py", line 17, in <module>
    with open('/deac/csc/khuriGrp/zhaok220/clustering/scripts/ga_parameters.yaml') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/deac/csc/khuriGrp/zhaok220/clustering/scripts/ga_parameters.yaml'
